failed: MultiNodeSample

https://jenkins.akka.io:8498/job/akka-nightly/2267/consoleFull

    [JVM-2] A MultiNodeSample
    [JVM-2] - must wait for all nodes to enter a barrier
    [JVM-1] - must wait for all nodes to enter a barrier
    [JVM-1] - must send to and receive from a remote node *** FAILED ***
    [JVM-1]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for pong
    [JVM-1]   at scala.Predef$.assert(Predef.scala:165)
    [JVM-1]   at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:353)
    [JVM-1]   at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:330)
    [JVM-1]   at akka.testkit.TestKit.expectMsg(TestKit.scala:738)
    [JVM-1]   at sample.multinode.MultiNodeSample$$anonfun$1$$anonfun$apply$mcV$sp$2$$anonfun$apply$mcV$sp$3.apply$mcV$sp(MultiNodeSample.scala:49)
    [JVM-1]   at akka.remote.testkit.MultiNodeSpec.runOn(MultiNodeSpec.scala:338)
    [JVM-1]   at sample.multinode.MultiNodeSample$$anonfun$1$$anonfun$apply$mcV$sp$2.apply$mcV$sp(MultiNodeSample.scala:45)
    [JVM-1]   at sample.multinode.MultiNodeSample$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(MultiNodeSample.scala:44)
    [JVM-1]   at sample.multinode.MultiNodeSample$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(MultiNodeSample.scala:44)
    [JVM-1]   at org.scalatest.Transformer$$anonfun$apply$1.apply$mcV$sp(Transformer.scala:22)

--- a/akka-samples/akka-sample-multi-node-scala/src/multi-jvm/scala/sample/multinode/MultiNodeSample.scala
+++ b/akka-samples/akka-sample-multi-node-scala/src/multi-jvm/scala/sample/multinode/MultiNodeSample.scala
@@ -46,7 +46,8 @@ class MultiNodeSample extends MultiNodeSpec(MultiNodeSampleConfig)
         enterBarrier("deployed")
         val ponger = system.actorSelection(node(node2) / "user" / "ponger")
         ponger ! "ping"
-        expectMsg("pong")
+        import scala.concurrent.duration._
+        expectMsg(10.seconds, "pong")
       }
 
       runOn(node2) {

